AnnService/src/Core/VectorIndex.cpp

```diff
@@ -486,6 +506,13 @@ VectorIndex::LoadIndex(const std::string& p_loaderFilePath, std::shared_ptr<Vect
             p_vectorIndex->BuildMetaMapping();
         }
     }
+
```

Comment:
Try to load quantizer earlier than LoadIndexData so that the DistanceCalcSelector can select correct distance function.